index.md

```diff
@@ -0,0 +1,175 @@
+# GTFS-Realtime Data Best Practices
+
+## Introduction
+
+These are recommended practices for describing real-time public transportation information in the [GTFS Realtime](https://gtfs.org/reference/realtime/v2/) data format.
+
+### Document Structure
+
+Recommended practices are organized into two primary sections
+
+* __[Practice Recommendations Organized by message](#practice-recommendations-organized-by-message):__ Recommendations are organized by message and field in the same order described in the official GTFS-Realtime reference.
+* __[Practice Recommendations Organized by Case](#practice-recommendations-organized-by-case):__ With particular cases, such as frequency-based service, practices may need to be applied across several messages and fields and even with the corresponding GTFS package. Such recommendations are consolidated in this section.

```

Comment:
Clarify this to "Use Case" based on the description?

Reply:
This text mirrors the existing GTFS best practices text at the top of:
https://gtfs.org/best-practices/

I've tried to clarify it a bit here.